AxonDeepSeg/models/download_model.py

```diff
@@ -1,19 +1,31 @@
 from AxonDeepSeg.ads_utils import *
 from pathlib import Path
 import shutil
-
+import argparse
+from argparse import RawTextHelpFormatter
 
 def download_model(destination = None):
+ 
+
     if destination is None:
-        sem_destination = Path("AxonDeepSeg/models/default_SEM_model")
-        tem_destination = Path("AxonDeepSeg/models/default_TEM_model")
+        sem_destination = Path(__file__).parent / "default_SEM_model"
```

Comment:
According to the test coverage analysis, these two lines are missed by the tests. @vs74 could you add an additional test that deals with this case?

<img width="1037" alt="Capture d’écran 2020-11-16 à 10 49 40" src="https://user-images.githubusercontent.com/1421029/99263208-76e52200-27f9-11eb-93b4-a38128a015a7.png">

Reply:
Well, the thing is that when destination is `None`, the models get saved in the deafult `AxonDeepSeg/models` directory. Adding another test to cover these lines, would add an extra overhead in Travis CI testing since these lines would be executed twice: first during pip install -e . and second during testing, which would overwrite the existing models stored in default directory, `AxonDeepSeg/models/`. Do you think it's a good idea to do that?